Makefile

```diff
@@ -68,3 +68,6 @@ endif
 # redirect all
 $(TARGETS):
 	$(MAKE) --directory=vmdk_plugin $@
+
+.DEFAULT:
```

Comment:
"default target" earlier is the target which is invoked by default  :-) (i.e. on `make` command, with no targets. `.DEFAULT:` is rules of last resort - executed when target is passed but not found in Makefile. Real good for fall through.

https://www.gnu.org/software/make/manual/html_node/Special-Targets.html. 
```
.DEFAULT
The recipe specified for .DEFAULT is used for any target for which no rules are found (either
explicit rules or implicit rules). See Last Resort. If a .DEFAULT recipe is specified, every file
mentioned as a prerequisite, but not as a target in a rule, will have that recipe executed on its
behalf. See Implicit Rule Search Algorithm.